api/src/main/java/org/openmrs/module/pihcore/PihCoreActivator.java

```diff
@@ -14,10 +14,10 @@
 package org.openmrs.module.pihcore;
 
 
+import org.apache.commons.lang.StringUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.openmrs.GlobalProperty;
-import org.openmrs.PatientIdentifierType;
+import org.openmrs.*;
```

Comment:
A nitpick, but we shouldn't be using global imports as it's OpenMRS convention not to... do you have IntelliJ doing this replacement?